ion_phys/tests/test_tss.py

```diff
@@ -14,6 +14,27 @@ def _steady_state_population(intensity):
 
 class TestTSS(unittest.TestCase):
     """two-state system tests"""
+    def test_spont_rates(self):
+        """Test the spontaneous rates satisfy relations in net rates"""
+        intensity_list = [1e-3, 1e-1, 0.3, 1, 1.0, 2, 10., 1.2e4]
+
+        ion = Ca43(B=5e-4, level_filter=[ground_level, P32])
+        s_idx = ion.index(ground_level, 4)
+        p_idx = ion.index(P32, +5)
```

Comment:
Might be worth a comment that we're using a cycling transition and sigma light to make a two state system (this isn't obvious at first otherwise).